### ISSUE TITLE
Générer des PDF à la volée à partir d'image

### ISSUE THREAD

**Issue by JBPressac** [Daniel-KM/ArchiveFolder#5]
Bonjour,
Est-ce que Archive Folder permet de créer une version PDF des images constituant un item Omeka ? Un des plugins que vous développez permet-il de le faire ? Merci

**Comment by Daniel-KM** [comment]
Non, cela ne fait pas cela. Le plus simple est de lancer un script maison sur tous les dossiers et de créer le pdf pour chaque dossier. Sinon, on peut mettre le même script dans un plugin et l'attacher à un hook "after save" et créer le fichier à ce moment-là.

**Comment by JBPressac** [comment]
Super, merci du conseil.
